roles/hosted_engine_setup/tasks/pre_checks/filter_unsupported_vlan_devices.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Check for VLAN devices
+  set_fact:
+    is_vlan: "{{ nic_if.stdout == 'vlan' }}"
+  when: nic_if.stdout.find('vlan') != -1
+  with_items:
+    - "{{ interface_types.results }}"
+  loop_control:
+    loop_var: nic_if
+  register: vlan_list
+- debug: var=vlan_list
+- name: Check for base interface of VLAN devices
+  command: nmcli -g VLAN.PARENT device show {{ vlan_device.nic_if.nic }}
```

Comment:
Can't we use `nmcli` module?

Reply:
unfortunately no, 'nmcli' module is to create/manage networks and give back only the status of each operation and not to return info about existing networks.